main_logic/core.py

```
@@ -2032,27 +2158,32 @@ async def feed_tts_chunk(self, text: str):
-            if self.session and hasattr(self.session, '_conversation_history'):
-                self.session._conversation_history.append(_AIMsg(content=full_text))
+                from utils.llm_client import AIMessage as _AIMsg
+                if self.session and hasattr(self.session, '_conversation_history'):
+                    self.session._conversation_history.append(_AIMsg(content=full_text))
 
-            if self.use_tts and self.tts_thread and self.tts_thread.is_alive():
+                if self.use_tts and self.tts_thread and self.tts_thread.is_alive():
+                    try:
+                        self.tts_request_queue.put((None, None))
+                    except Exception:
+                        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)